lib/amorail/entities/contact.rb

```diff
@@ -1,43 +1,36 @@
 module Amorail
   class AmoContact < Amorail::AmoEntity
-    set_amo_name "contacts"
+    set_amo_name 'contacts'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.